src/PlcNodeManager.cs

```diff
@@ -186,7 +186,7 @@ private void AddSpecialNodes(FolderState dataFolder)
         {
             if (PlcSimulation.AddSpecialCharName)
             {
-                const string SpecialChars = "\"!�$%&/()=?`�\\+~*'#_-:.;,<>|@^���{[]}";
+                const string SpecialChars = "\"!�$%&/()=?`�\\+~*'#_-:.;,<>|@^���{[]}";

```

Comment:
Pls revert this change

Reply:
When we are at it:
I always get an exception: _plcServer.PlcNodeManager.SpecialCharNameNode.Stop(); (133)
when I close the program.

Reply:
Could you do me a favor and change the following:
In PlcNodeManager to avoid this type of encoding issues:
string SpecialChars = HttpUtility.HtmlDecode(@"&quot;!&#167;$%&amp;/()=?`&#180;\+~*&#39;#_-:.;,&lt;&gt;|@^&#176;€&#181;{[]}");

In PlcSimulation, it should fix the Stop error:
_plcServer.PlcNodeManager.SpikeNode?.Stop();
_plcServer.PlcNodeManager.DipNode?.Stop();
_plcServer.PlcNodeManager.PosTrendNode?.Stop();
_plcServer.PlcNodeManager.NegTrendNode?.Stop();
_plcServer.PlcNodeManager.AlternatingBooleanNode?.Stop();
_plcServer.PlcNodeManager.StepUpNode?.Stop();
_plcServer.PlcNodeManager.RandomSignedInt32?.Stop();
_plcServer.PlcNodeManager.RandomUnsignedInt32?.Stop();
_plcServer.PlcNodeManager.SpecialCharNameNode?.Stop();

Thanks!